ts_scripts/install_dependencies.py

```diff
@@ -1,8 +1,8 @@
+import argparse
```

Comment:
I don't think conda build is related to this file. Pls leave this file as it is if there are no changes.

Reply:
The lint tool is auto-updating this file.

Reply:
Reverted this file